ports/unix/coverage.c

```diff
@@ -692,6 +692,24 @@ static mp_obj_t extra_coverage(void) {
         ringbuf.iget = 0;
         ringbuf_put(&ringbuf, 0xaa);
         mp_printf(&mp_plat_print, "%d\n", ringbuf_get16(&ringbuf));
+
+        // ringbuf_put_bytes() / ringbuf_get_bytes() functions.
+        ringbuf.iput = 0;
+        ringbuf.iget = 0;
+        uint8_t *put = (uint8_t *)"abc123";
+        uint8_t get[7] = {0};
+        mp_printf(&mp_plat_print, "%d\n", ringbuf_put_bytes(&ringbuf, put, 7));
+        mp_printf(&mp_plat_print, "%d\n", ringbuf_get_bytes(&ringbuf, get, 7));
+        mp_printf(&mp_plat_print, "%s\n", get);
+        // Prefill ringbuffer
+        for (int i = 0; i < 98; ++i) {
```

Comment:
Instead of 98, maybe `sizeof(buf) - 2`?  Or even `sizeof(buf) - 3` to indicate that you allow 2 bytes free space, then the 7 added below will not fit.